pkg/schemaconv/testdata/new-schema.yaml

```diff
@@ -229,7 +229,7 @@ types:
         scalar: string
     - name: data
       type:
-        namedType: io.k8s.apimachinery.pkg.runtime.RawExtension
```

Comment:
I notice that we're not removing the type though, maybe we should drop it altogether?

Reply:
Can we leave the name here and just make it declare a thing that is untyped?

Reply:
(my original plan was to make the schema set the "guess at keys" flag for these)

Reply:
/shrug
I don't really have a strong opinion.

Reply:
"these" being  `untyped` or `RawExtension`?

Reply:
Specifically, RawExtension maps to Untyped{GuessAtNames: true}.